code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -943,6 +943,7 @@ nobiliumsuppression = INFINITY
 		return NO_REACTION
 	location.rad_act(produced_amount * 2.4)
 	cached_gases[/datum/gas/tritium][MOLES] -= produced_amount
+	ASSERT_GAS(/datum/gas/hydrogen, air)
```

Comment:
For cleanliness's sake, assert the gas after the rad act and before we start doing anything with the gaslists

Reply:
Done